Extensions/xkit_patches.js

```diff
@@ -1853,14 +1862,21 @@ XKit.tools.dump_config = function(){
 			if (typeof XKit.page.peepr != "undefined" && XKit.page.peepr === true) {
 				post_count = $(".post").length;
 			} else {
-				if ($("#posts").length === 0) {
-					return;
```

Comment:
took out early returns since function should fully run on pages with no posts but sideview could appear